packages/world-module-erc20/ts/defineERC20Module.ts

```diff
@@ -1,12 +1,13 @@
 import { encodeAbiParameters, stringToHex } from "viem";
+import { Module } from "@latticexyz/world/config/v2";
 
-export type DefineERC20ConfigInput = {
+export type DefineERC20ModuleInput = {
   namespace: string;
   name: string;
   symbol: string;
 };
 
-export function defineERC20Config({ namespace, name, symbol }: DefineERC20ConfigInput) {
+export function defineERC20Module({ namespace, name, symbol }: DefineERC20ModuleInput): Module {
```

Comment:
this should prob use `ModuleInput` as its being used as input for the config 

```suggestion
export function defineERC20Module({ namespace, name, symbol }: DefineERC20ModuleInput): ModuleInput {
```